doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2323,6 +2323,18 @@ Printing universes
 Existential variables
 ---------------------
 
+.. insertgram term_evar inst
+
+.. productionlist:: coq
+   term_evar : ?[ `ident` ]
+             : ?[ ?`ident` ]
+             : ?`ident` `evar_instance`
+   evar_instance : @{ `inst_list_semi` }
+                 : `empty`
+   inst_list_semi : `inst_list_semi` ; `inst`
+                  : `inst`
+   inst : `ident` := `term`
```

Comment:
We should try to find a better name for `inst`. @herbelin: a suggestion?

Reply:
I'm afraid I have no good suggestion, maybe `var_def`, or `binding`. Somehow, this could be extended to the same syntax as in record fields (i.e. `ident binders := term`) and unified with `field_def`. On the other side, the name `field` comes with a specific connotation which we would lose in a merge.

Reply:
How about `evar_binding`?

Reply:
Yes, why not.